### DISCUSSION TITLE
Welcome to Luigi Discussions!

### DISCUSSION THREAD

**Discussion by alexandra-simeonova** [SAP/luigi#2637] [General]
## 👋 Welcome!
  We're using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you're wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

In addition to Discussions, check out these other resources:
* [Luigi Documentation](https://docs.luigi-project.io/docs)
* [Luigi FAQ ](https://docs.luigi-project.io/docs/faq)
* [Luigi Slack Channel](https://slack.luigi-project.io/)
* [Luigi Blog / Release notes](https://luigi-project.io/blog/overview)

If you would like to contribute to Luigi, take a look at our: 
* [Contribution guidelines](https://github.com/SAP/luigi/blob/master/CONTRIBUTING.md)
* [Documentation guidelines](https://github.com/SAP/luigi/blob/master/docs/content-guidelines.md)

**Comment by gtb104** [comment]
Is the Slack channel still active? How do you get a logon to join the discussion?

**Reply by alexandra-simeonova** [reply]
Hello, yes, the Slack channel at https://luigi-project.slack.com/ is still active. To join it, you need to create a Slack account or sign in with your existing Slack account. 
You can see the Slack website for more info: https://slack.com/help/articles/212681477-Sign-in-to-Slack

**Reply by gtb104** [reply]
I try to sign in to the Luigi Slack channel and I get an email saying that I don't have an account on Luigi tied to the email I'm using to sign into Slack.

**Reply by alexandra-simeonova** [reply]
Hi, can you try to join through this link instead: https://slack.luigi-project.io/
If this doesn't work, please tell us your email so we can add you manually.

**Reply by gtb104** [reply]
That URL worked. I'm in! Thanks!
